djangocms_audio/locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,95 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-07-26 16:27+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: cms_plugins.py:12 cms_plugins.py:39 cms_plugins.py:68
+msgid "Audio Player"
+msgstr ""
+
+#: cms_plugins.py:23 cms_plugins.py:50 cms_plugins.py:78
+msgid "Advanced Settings"
+msgstr ""
+
+#: cms_plugins.py:38 models.py:88
+msgid "File"
+msgstr ""
+
+#: cms_plugins.py:67 models.py:146
+msgid "Folder"
+msgstr ""
+
+#: models.py:53
+msgid "Default"
+msgstr ""
+
+#: models.py:58
+msgid "Template"
+msgstr ""
+
+#: models.py:64
+msgid "Label"
+msgstr ""
+
+#: models.py:69
+msgid "Is active"
+msgstr ""
+
+#: models.py:74 models.py:109 models.py:153
+msgid "Attributes"
+msgstr ""
+
+#: models.py:95
+msgid "Title"
+msgstr ""
+
+#: models.py:100
+msgid "Description"
+msgstr ""
+
+#: models.py:104
+msgid "Text transcript"
+msgstr ""
+
+#: models.py:106
+msgid "Provide a text transcript for accessibility support."
+msgstr ""
+
+#: models.py:122
+#, python-format
+msgid "Incorrect file type: %s."
+msgstr ""
+
+#: models.py:130
+msgid "<file is missing>"
+msgstr ""
+
+#: models.py:155
+msgid "Applied to all audio file instances."
```

Comment:
Is applied? Will be applied? Needs to be clear what this means in the interface.